ethabi/src/decoder.rs

```diff
@@ -528,6 +633,81 @@ mod tests {
 		);
 	}
 
+	#[test]
+	fn decode_params_containing_dynamic_tuple() {
```

Comment:
It would be nice to have a test for decoding and a test for encoding a nested tuple.